tests/deploy/e2e/test_autostarted.py

```diff
@@ -412,12 +412,15 @@ def find_scheduler(children: list[Process]) -> Optional[Process]:
         current_scheduler = None
 
         for child in children:
+            # ignore zombie children
+            if child.status() == psutil.STATUS_ZOMBIE:
+                continue
```

Comment:
This probably won't change a lot since calling `cmdline()` on a zombie process still returns something e.g. `{ZombieProcess}PID still exists but it's a zombie (pid=12844, name='pg_ctl')` on linux, but it can't hurt either I think.